src/main/java/com/jagrosh/jmusicbot/queue/FairQueue.java

```diff
@@ -15,20 +15,27 @@
  */
 package com.jagrosh.jmusicbot.queue;
 
-import java.util.ArrayList;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Set;
 
 /**
  *
  * @author John Grosh (jagrosh)
  * @param <T>
  */
-public class FairQueue<T extends Queueable> {
-    private final List<T> list = new ArrayList<>();
-    private final Set<Long> set = new HashSet<>();
-    
+public class FairQueue<T extends Queueable> extends AbstractQueue<T> {
+
+    public FairQueue(AbstractQueue<T> queue) {
+        super(queue);
+    }
```

Comment:
Put the curly braces here in a new line.
```suggestion
public class FairQueue<T extends Queueable> extends AbstractQueue<T>
{

    public FairQueue(AbstractQueue<T> queue)
    {
        super(queue);
    }
```